README.md

```diff
@@ -108,6 +115,25 @@ Using this workflow you can build the test suite image locally, load the image i
 
 Using make is not required, feel free to peak inside the Makefile in order to call the test driver directly for whatever workflow you may need.
 
+
+### Debugging failures
+
+If you want to get the logs from any process started as part of running tests (the ceramic node, the CAS, the test output itself), you can use:
+`kubectl logs -n <namespace> <pod name>`
+
+You can see a list of running kubernetes pods and their namespace and pod name by running `k9s`.
+
+
+### Cleaning up
+
+To tear down all the pods created by running the tests, you can delete the network that they were set up in.
+
+    > kubectl get networks # Look up the network name
+    NAME                          AGE
+    smoke-basic-go-rust-dev-run   33m
+
+    > kubectl delete network smoke-basic-go-rust-dev-run # delete the network and all pods within it.
```

Comment:
Should we also delete the namespace? It doesn't get deleted when the network is deleted, I believe.

Reply:
🤷 Don't feel strongly.  I haven't yet had a need to delete the namespace, but having a handy one-liner to kill all the running pods was useful